crates/re_viewer/src/ui/welcome_screen/welcome_page.rs

```diff
@@ -324,3 +346,32 @@ fn open_markdown_recording(
 
     Ok(())
 }
+
+/// The User-Agent of the user's browser.
+fn user_agent() -> Option<String> {
+    #[cfg(target_arch = "wasm32")]
+    let result = web_sys::window()?.navigator().user_agent().ok();
+
+    #[cfg(not(target_arch = "wasm32"))]
+    let result = None;
+
+    result
+}
```

Comment:
```suggestion
    #[cfg(target_arch = "wasm32")]
    eframe::web::user_agent()

    #[cfg(not(target_arch = "wasm32"))]
    None
}
```

Reply:
That doesn't work.

<img width="602" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/cd41dd33-630b-4ae9-b4c5-35036e982746">

Reply:
This is the most compact I can think of:
```rust
fn user_agent() -> Option<String> {
    #[cfg(target_arch = "wasm32")]
    return eframe::web::user_agent();

    #[cfg(not(target_arch = "wasm32"))]
    None
}
```